tests/test_timestepping.py

```diff
@@ -15,28 +15,28 @@ def grid(shape=(11, 11)):
 def a(grid):
     """Forward time data object, unrolled (save=True)"""
     return TimeFunction(name='a', grid=grid, time_order=1,
-                        time_dim=6, save=True)
+                        save=6)
```

Comment:
Undo indent? (same for fixtures `b` and `d` below)

Reply:
Fixed